index.html

```diff
@@ -3,53 +3,188 @@
   <head>
     <title>Girma's Page</title>
     <meta charset="UTF-8" />
-    <meta name="keywords" content="HTML, CSS, JavaScript" /> 
-    <link rel="stylesheet" type="text/css" href="css/index.css"> 
-    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto">
+    <link rel="stylesheet" type="text/css" href="css/index.css" />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css?family=Roboto"
+    />
   </head>
   <body>
     <nav>
       <ul>
         <li><a href="#About">About</a></li>
         <li><a href="#Experience">Experience</a></li>
+        <li><a href="#skills">Skills</a></li>
+        <li><a href="#projects">Projects</a></li>
         <li><a href="#Connect">Connect</a></li>
       </ul>
     </nav>
     <section id="About">
-      <div class="Wrapper">
-        <h1>Girma Ebssa</h1>
-        <img class = "img-coding" src="image/code.jpg" alt="How Girma practice coding">
-        
+      <h2>Girma Ebssa</h2>
+      <div class="image-container">
+        <img
+          src="image/code.jpg"
+          alt="How Girma practice coding"
+          class="profile-image"
+        />
       </div>
+      <p>
+        Girma Ebssa has completed basic courses in programing, javascript, html
+        and css through RefCode. I have also completed bubble bootcamp course to
+        develop a web using no code platform approach from RefCode by
+        volunteers. A Bubble Bootcamp is a dedicated program that teaches you
+        how to build web apps using the powerful no-code platform called Bubble.
+        These bootcamps are designed to help you master Bubble, regardless of
+        your starting point. They cover various aspects of app development, and
+        you’ll learn how to create web applications without writing code.
+        Additionally, I am currently enrolled in the Io-programing course at
+        CodeTheDream. CodeTheDream offers free code school programs and paid
+        apprenticeships, providing real-world experience in app development for
+        a diverse range of clients.
+      </p>
+    </section>
+
+    <!-- <section id="About">
+      <h1>Girma Ebssa</h1>
+      <div class="img-container">
+        <img src="image/code.jpg" alt="How Girma practice coding" />
+      </div>
+
       <h2>About</h2>
```

Comment:
Since this About header is commented out, none of the CSS given in the stylesheet is being applied to it.